README.md

```diff
@@ -45,10 +46,10 @@ sonarqube6:SonarQubeConfiguration sonarqubeConfig = {
     }
 };
    
-sonarqube6:Client sonarqube = new(sonarqubeConfig);
+sonarqube6:Client sonarqubeClient = new(sonarqubeConfig);
 
-public function main() {
-   var projectDetails = sonarqube->getProject(config:getAsString(PROJECT_NAME));
+public function main(string... args) {
+   var projectDetails = sonarqubeClient->getProject(config:getAsString(PROJECT_NAME));
    if (projectDetails is sonarqube6:Project) {
        io:println(projectDetails)
    } else {
```

Comment:
The following line fails right? `io:println` doesn't take a defaultable param. Shall we change to 
```ballerina
       io:println("Error: ", projectDetails);
```